client/src/components/Header.tsx

```diff
@@ -18,11 +18,19 @@ export function Header() {
     <>
       <div>
         <div className="p-2 px-[15px] flex justify-between">
-          <h1 className="text-[32px] font-bold">NewU</h1>
+          <Link
+            to="/"
+            className="text-[32px] font-bold cursor-pointer"
+            style={{ fontFamily: 'Arvo, serif' }}>
+            NewU
+          </Link>
           {user && (
             <div className="flex items-center">
-              <span className="h-fit mr-[20px]">{user?.displayName}</span>
+              <h1 className="h-fit mr-[20px] text-lg inline-block">
+                {user?.displayName}
```

Comment:
Because of the check for `user` up above, you know that `user` is defined here, so the `?` is unnecessary.